ui/package.json

```diff
@@ -15,6 +15,7 @@
     "object-assign-deep": "^0.4.0",
     "query-string": "^7.0.1",
     "react": "16.14.0",
+    "react-collapsed": "^3.0.1",
```

Comment:
That's a brilliant suggestion. Will take a look into this

Reply:
`ExpandableContainer` and `CollapsibleBadgeGroup ` are conceptually different (first is to hide the content if it exceeds provided max heights, where second is a regular `EuiBadgeGroup`, but with extra logic to only render `X` badges and hide the rest behind the collapsed view). So, we can't replace `ExpandableContainer` with `CollapsibleBadgeGroup`.

As for the suggestion to refactor `ExpandableContainer` with `react-collapsed`, it should be possible to do

Reply:
It's done here: https://github.com/gojek/turing/pull/89/commits/08ca3d4de41e3dcf19d22eaffd656594d0aa717d. 

It was  less straightforward than I expected, because of the extra logic that we have in `ExpandableContainer` that only makes it possible to expand/collapse if the content of the container exceeds the value of `maxCollapsedHeight`.